src/main/java/uk/gov/hmcts/ccd/auditlog/AuditOperationType.java

```diff
@@ -11,7 +11,8 @@
     REMOVE_CASE_ASSIGNED_USER_ROLES("Remove Case-Assigned Users and Roles"),
     GET_CASE_ASSIGNED_USER_ROLES("Get Case-Assigned Users and Roles"),
     REVOKE_CASE_ACCESS("Revoke case access permissions"),
-    VIEW_CASE_HISTORY("View case history");
+    VIEW_CASE_HISTORY("View case history"),
+    GLOBAL_SEARCH("GlobalSearch");
```

Comment:
can we inline like others "Global search"

Reply:
fixed in [cfb1298](https://github.com/hmcts/ccd-data-store-api/pull/1672/commits/cfb129896c8f8d8d4c56b4f0af2222221d6c5474)